app/options.ts

```diff
@@ -77,6 +82,23 @@ export const options: {
 			},
 		],
 	},
+	sortingOrder: {
+		since: '1.15.0',
```

Comment:
Looks like the last version is 1.7.1 right? So 1.8.0 would be the next version in which this is released probably